### PR TITLE
add docs link in airflow message banner on database add new service page.

### PR DESCRIPTION
**Pull request by Abhishek332**
**### Before**
Screenshot 2024-01-04 at 4 06 23 PM

**### After** 
Screenshot 2024-01-04 at 4 04 43 PM

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/ServiceConfig/ConnectionConfigForm.tsx [hunk 1]
```diff
@@ -154,7 +154,7 @@ const ConnectionConfigForm: FunctionComponent<Props> = ({
 
   return (
     <Fragment>
-      <AirflowMessageBanner />
+      <AirflowMessageBanner docLink="https://docs.open-metadata.org/v1.2.x/deployment/ingestion/openmetadata#troubleshooting" />
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ServiceConfig/ConnectionConfigForm.tsx:157]: Move this link to doc constants.

File: openmetadata-ui/src/main/resources/ui/src/components/common/AirflowMessageBanner/AirflowMessageBanner.tsx [hunk 2]
```diff
@@ -32,7 +40,16 @@ const AirflowMessageBanner: FC<SpaceProps> = ({ className }) => {
...
+      <Typography.Text>
+        {docLink ? (
+          <>
+            {reason?.endsWith('.') ? reason?.slice(0, -1) : reason}{' '}
+            <a href={docLink}>{t('label.here-lowercase')}.</a>
+          </>
+        ) : (
+          reason
+        )}
+      </Typography.Text>
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/AirflowMessageBanner/AirflowMessageBanner.tsx:52]: @Abhishek332 , let's write the unit test for this.

File: openmetadata-ui/src/main/resources/ui/src/components/common/AirflowMessageBanner/AirflowMessageBanner.tsx [hunk 3]
```diff
@@ -32,7 +40,16 @@ const AirflowMessageBanner: FC<SpaceProps> = ({ className }) => {
       data-testid="no-airflow-placeholder"
       size={16}>
       <IconRetry className="align-middle" height={24} width={24} />
-      <Typography.Text>{reason}</Typography.Text>
+      <Typography.Text>
+        {docLink ? (
+          <>
+            {reason?.endsWith('.') ? reason?.slice(0, -1) : reason}{' '}
+            <a href={docLink}>{t('label.here-lowercase')}.</a>
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/AirflowMessageBanner/AirflowMessageBanner.tsx:47]: Can we use the `<Link>` here and pass the target="_blank", so it should open in a new tab?

**Review by Sachin-chaurasiya — COMMENTED** [review]
Left a few comments, thanks

**Comment by chirag-madlani** [comment]
putting this to HOLD, cc: @ulixius9 @pmbrull

**Comment by chirag-madlani** [comment]
Closing since we are mostly handling it from the backend. Ticket created here: https://github.com/open-metadata/OpenMetadata/issues/14590

**Pull request closed without merging**